### PR TITLE
CPFA-38 navBar fixed animation bug

### PR DESCRIPTION
**Pull request by KenAdams777**
At the start of the animation, 'My profile' was separated into two different lines because of the initial width.

### CODE REVIEW

File: src/components/Menu/Menu.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function Menu({ isMenuOpen }) {
 						<Link to="/bookmarks">Bookmarks</Link>
 					</li>
 					<li>
-						<Link to="/my-profile">My Profile</Link>
+						<Link to="/my-profile">My&nbsp;Profile</Link>
```
Comment (alexiStructShare) [inline src/components/Menu/Menu.js:22]: Why this is the fix? Can you please add screenshots with problem to the trello ticket?

Comment (KenAdams777) [reply src/components/Menu/Menu.js:22]: > Why this is the fix? Can you please add screenshots with problem to the trello ticket?
Ok. I'll add a screenshot, also I described the bug in the pull request description.

Comment (KenAdams777) [reply src/components/Menu/Menu.js:22]: > Why this is the fix? Can you please add screenshots with problem to the trello ticket?

Ok. I'll add a screenshot, also I described the bug in the pull request description.

Comment (MuscularSloth) [reply src/components/Menu/Menu.js:22]: I think `white-space: nowrap;` can help you here

Comment (KenAdams777) [reply src/components/Menu/Menu.js:22]: > I think `white-space: nowrap;` can help you here

This approach also works, so I think no reason to change. Thx, for the good suggestion, will use it next time.

**Review by RudeAnna — APPROVED** [review]
Now I see this can be fixed with css without adding html symbol &nbsp; like Konstantin suggested, but this way it still works

File: src/components/Menu/Menu.js (hunk 1 above)
Comment (MuscularSloth) [reply src/components/Menu/Menu.js:22]: If we will get this info from some kind of object the bets solution do it with css rule. I think its more universal.

Comment (KenAdams777) [reply src/components/Menu/Menu.js:22]: > If we will get this info from some kind of object the bets solution do it with css rule. I think its more universal.

Good point, but still in our case it's not gonna be like you say, so I think it won't cause problems so we can leave it as it is.

**Review by MuscularSloth — APPROVED** [review]
Good job! =)

**Pull request merged**